src/components/markdown.js

```diff
@@ -26,6 +26,18 @@ const Markdown = props => {
     }
     return ""
   }
+  const SectionTitle = props => {
+    if (props.label) {
+      return (
+        <>
+          {`<h3 align="left">${props.label}</h3>`}
+          <br />
+          <br />
```

Comment:
I guess we can remove one `br`
<img width="370" alt="Screenshot 2020-09-25 at 5 18 26 PM" src="https://user-images.githubusercontent.com/26406279/94263821-48626d80-ff53-11ea-989d-717957737e35.png">

`br` tag is adding an extra line in the preview